sitecore-packages.json

```diff
@@ -1,231 +1,235 @@
 {
```

Comment:
Seems like you accidentally changed the spacing in this file?

Reply:
The spacing changed when I ran the `.\contributing\Sort-SitecorePackagesJson.ps1` script. I can manually fix it now.

Reply:
The spacing issue has been fixed now. Thank you for reporting it.